donkeycar/pipeline/training.py

```diff
@@ -81,6 +81,14 @@ def create_tf_data(self) -> tf.data.Dataset:
             output_shapes=self.model.output_shapes())
         return dataset.repeat().batch(self.batch_size)
 
+    def create_tf_data(self) -> tf.data.Dataset:
```

Comment:
Please remove, this method is already defined 8 lines above.

Reply:
So sorry about this.